src/components/PrimaryNav/_primary-nav.scss

```diff
@@ -91,6 +84,30 @@
     }
   }
 
+  .p-primary-nav__bottom {
```

Comment:
The spacing around the horizontal rule is not the same. `p-list is-internal` could use less bottom margin.

<img width="247" alt="Screen Shot 2020-01-22 at 11 10 41 AM" src="https://user-images.githubusercontent.com/532033/72916625-f902d800-3d07-11ea-869e-d9f0f77da771.png">